app/Main.hs

```diff
@@ -99,13 +123,166 @@ makePscPackage force = do
   where
     pscPackageJsonPath = T.fromText "psc-package.json"
 
+-- | Copies over `spacchetti.dhall` to set up a Spacchetti project
+makeSpacchetti :: Bool -> IO ()
+makeSpacchetti force = do
+  T.unless force $ do
+    hasSpacchettiDhall <- T.testfile spacchettiDhallPath
+    T.when hasSpacchettiDhall $ T.die
+       $ "Found " <> unsafePathToText spacchettiDhallPath <> ": there's already a project here. "
+      <> "Run `spacchetti initSpacchetti --force` if you're sure you want to overwrite it."
+  T.touch spacchettiDhallPath
+  T.writeTextFile spacchettiDhallPath Templates.spacchettiDhall
+  T.void $ T.shell ("dhall format --inplace " <> spacchettiDhallText) T.empty
```

Comment:
Let's depend on `dhall` so that we don't have to have it installed (way less friction for the user)

Reply:
Sounds horrible for our build times, and the other commands already use dhall executables. You'll have to depend on both Dhall and Dhall-JSON if you want to do this

Reply:
I think we don't strictly need `dhall-json`, but anyways build-times is only a CI problem?